test/unit/config/test__base.py

```diff
@@ -357,7 +357,7 @@ def test_containerization_unsafe_write_setting(tmp_path, runtime, mocker):
 
     expected = {
         'docker': None,
-        'podman': 1,
+        'podman': '1',
     }
 
-    assert rc.env.get('ANSIBLE_UNSAFE_WRITES') == expected[runtime]
+    assert str(rc.env.get('ANSIBLE_UNSAFE_WRITES')) == expected[runtime]
```

Comment:
I think it should be
```suggestion
    assert rc.env.get('ANSIBLE_UNSAFE_WRITES') == expected[runtime]
```
(otherwise a string would never equal `None` when the env var is missing)